storage/country.cpp

```diff
@@ -100,15 +122,21 @@ class DoStoreCountries
 class DoStoreFile2Info
 {
   map<string, CountryInfo> & m_file2info;
-  string m_lastFlag;
+  int64_t const m_version;
 
 public:
-  DoStoreFile2Info(map<string, CountryInfo> & file2info) : m_file2info(file2info) {}
+  DoStoreFile2Info(map<string, CountryInfo> & file2info, int64_t version)
+    : m_file2info(file2info), m_version(version) {}
 
-  void operator()(string name, string file, string const & flag, uint32_t mapSize, uint32_t, int)
+  void operator()(string name, string file, uint32_t mapSize, uint32_t, int)
   {
-    if (!flag.empty())
-      m_lastFlag = flag;
+    if (version::IsSingleMwm(m_version))
```

Comment:
More or less. Look here. For the time being we support two different countries.txt formats. Old and new one. So there're two different rules for parsing it. If we remove the "if (version::IsSingleMwm(m_version))" we stop supporting old countries.txt format.

You can raise an objection: we do not need an old countries.txt at all. We need to support a new one and support working with big (two components) mwms. 
Yes, I'll devote to it one of next PR but now I prefer to leave ability of parsing two countries.txt.

Yura, if you have further questions concerning this matter probably it's better to discuss it by voice.